service/src/main/java/uk/gov/hmcts/reform/fpl/enums/interfaces/LabelledAssignee.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.reform.fpl.enums.interfaces;
+
+public interface LabelledAssignee {
```

Comment:
minor, i have the feeling that Assignee or Labelled are better name from business perspective

Reply:
Good point, I named it Labelled at first to reflect what it represents, but Assignee sounds a little bit less confusing. Changed.